.github/workflows/on-code-change.yml

```diff
@@ -0,0 +1,60 @@
+# This workflow is triggered every time a change is pushed to any branches 
+# Github actions command reference: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+name: On Code Change (PR)
+
+# The workflow could also be triggered on PRs
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  update-signature:
+    name: Update module signature
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Can you use the new version of the actions, this way we don't get the depreciation warning (here and in other places in the PR)